base/poco/Foundation/include/Poco/Platform.h

```diff
@@ -212,25 +212,6 @@
 #endif
 
 
-#if defined(__clang__)
-#    define POCO_COMPILER_CLANG
```

Comment:
... not needed because the only usage was in `base/poco/Foundation/include/Poco/Alignment.h`

Reply:
... true